### PR TITLE
Migrate to Stitches

### PR DESCRIPTION
**Pull request by luixo**

### CODE REVIEW

**Review by luixo — COMMENTED** [review]
@frol @shelegdmitriy 
Please go through stage thoroughly as we don't have visual tests.
I did my best at comparing current version and upcoming one, but there's no chance everything's perfect.

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: frontend/src/components/nodes/__test__/__snapshots__/NodesEpoch.test.tsx.snap [hunk 1]
```diff
@@ -109,154 +109,58 @@ exports[`<NodesEpoch /> renders 1`] = `
...
         preserveAspectRatio="none"
         style={Object {}}
         viewBox="0 0 100 1"
       >
         <path
           className="rc-progress-line-trail"
-          d="M 0.5,0.5
-         L 99.5,0.5"
+          d="M 0,0.5
+         L 100,0.5"
```
[the remark this answers is not in the document]
Comment (shelegdmitriy) [reply frontend/src/components/nodes/__test__/__snapshots__/NodesEpoch.test.tsx.snap:140]: I didn't see any difference, so we may skip it

File: frontend/src/components/nodes/__test__/__snapshots__/NodesEpoch.test.tsx.snap [hunk 2]
```diff
@@ -77,7 +77,7 @@ exports[`<NodesEpoch /> renders 1`] = `
    a 48,48 0 1 1 0,-96"
               fillOpacity="0"
               stroke="transparent"
-              strokeLinecap="round"
+              strokeLinecap="square"
```
[the remark this answers is not in the document]
Comment (shelegdmitriy) [reply frontend/src/components/nodes/__test__/__snapshots__/NodesEpoch.test.tsx.snap:80]: Agree, let's skip it

File: frontend/src/components/dashboard/DashboardBlock.tsx [hunk 3]
```diff
@@ -10,17 +10,13 @@ import Link from "../utils/Link";
...
-
-const DashboardBlock = ({ className }: Props) => {
+const DashboardBlock: React.FC = () => {
   const { t } = useTranslation();
   return (
     <DatabaseConsumer>
       {(context) => (
         <DashboardCard
-          className={`block-card ${className || ""}`}
+          className="ml-md-4"
```
[the remark this answers is not in the document]
Comment (luixo) [reply frontend/src/components/dashboard/DashboardBlock.tsx:20]: To distinct it from other dashboard block in e2e tests.

File: frontend/src/components/accounts/__tests__/__snapshots__/AccountRow.test.tsx.snap [hunk 4]
```diff
@@ -5,86 +5,30 @@ exports[`<AccountRow /> renders with long name 1`] = `
   onClick={[Function]}
 >
   <a
+    className="c-LinkWrapper-cIdiJW"
```
[the remark this answers is not in the document]
Comment (luixo) [reply frontend/src/components/accounts/__tests__/__snapshots__/AccountRow.test.tsx.snap:8]: I'm not sure, it's either `class` or `composer` or something like that.
Source: https://github.com/modulz/stitches/blob/59418ebb65f351d9e3bbd4cce211efde24fd9479/packages/core/src/features/css.js#L71

File: frontend/src/components/utils/DashboardCard.tsx [hunk 5]
```diff
@@ -1,73 +1,73 @@
...
   iconPath,
   title,
   headerRight,
+  className,
+  dataId,
   children,
 }: Props) => (
-  <Row noGutters className={`dashboard-card ${className || ""}`}>
+  <DashboardCardWrapper noGutters className={className} data-id={dataId}>
+    <div data-id="foo" />
```
[the remark this answers is not in the document]
Comment (luixo) [reply frontend/src/components/utils/DashboardCard.tsx:57]: Sorry, didn't remove my test cases :)

File: frontend/src/components/contracts/ContractDetails.tsx [hunk 6]
```diff
@@ -59,17 +96,14 @@ const ContractDetails: FC<Props> = ({ accountId }) => {
...
+        <CardCellTitleImage src={"/static/images/icon-d-contract.svg"} />
         {t("common.contracts.contract")}
-      </div>
-      <div className="contract-info-container">
+      </ContractTitle>
+      <ContractInfoContainer>
         <Row noGutters className="border-0">
           <Col md="4">
-            <CardCell
+            <LockedCell
```
[the remark this answers is not in the document]
Comment (luixo) [reply frontend/src/components/contracts/ContractDetails.tsx:106]: That's a good catch.
Actually, `LockedCell` isn't needed at all, all it's styles are overrided deeper in the tree.

File: frontend/src/components/nodes/NodesCard.tsx [hunk 7]
```diff
@@ -7,8 +7,56 @@ import { useTranslation } from "react-i18next";
...
+  borderRadius: 8,
+  padding: "48px 32px",
+  marginTop: 50,
+
+  "@media (max-width: 768px)": {
+    marginTop: 32,
+    padding: "8px 16px 16px",
+    [`& ${InfoCardText}`]: {
+      fontSize: 20,
+    },
```
[the remark this answers is not in the document]
Comment (luixo) [reply frontend/src/components/nodes/NodesCard.tsx:31]: There is some misunderstanding here, I believe.
The `stats` page (the one from screenshots) is the same in terms of font size as it is in production.
The `nodes/validators` page (the one from lines you commented) have mentioned font sizes rules applied (you can check green validators amount, for example).

**Comment by luixo** [comment]
PR is rebased on "providers as hooks" commit, all notes are fixed, ready for you approves 🎉🎉🎉

**Comment by luixo** [comment]
@shelegdmitriy Please mark PR as approved or clarify problems that were not fixed.

**Review by shelegdmitriy — APPROVED** [review]

**Review by frol — APPROVED** [review]

**Pull request merged**